jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaEntrypointConstructor.java

```diff
@@ -37,6 +38,15 @@
         mainClass);
   }
 
+  /**
+   * Constructs the container entrypoint for the gcr.io/distroless/jetty base image.
+   *
+   * @return ["java", "-jar", "/jetty/start.jar"]
+   */
+  public static List<String> makeDistrolessJettyEntrypoint() {
```

Comment:
Should this take jvmFlags?

Reply:
Also, the `distroless/java/jetty` image has:
  - `entrypoint: ["/usr/bin/java", "-jar"]`
  - `cmd: ["/jetty/start.jar"]`

So we could just not set anything by default, right?

(As compared to `distroless/java` which sets `entrypoint: ["/usr/bin/java", "-jar"]` and no `cmd`, so we must change the entrypoint.)

Reply:
Sounds good if we are propagating both `Entrypoint` and `Cmd`. I think we do?

Reply:
Hmm, [it doesn't look like we do](https://github.com/GoogleContainerTools/jib/blob/master/jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/BuildImageStep.java#L118).